### DISCUSSION TITLE
Updated immich on my server but now immich is not loading and refreshes the page in a loop

### DISCUSSION THREAD

**Discussion by n4tur4lb0rnch1ll4** [immich-app/immich#9476] [Q&A]
Hi everybody,

just updated my immich server but now the page isn't showing anything and is reloading the page in a loop - does anyone know why and how to solve it?

Thank you very much and best regards,

Flo

**Comment by bo0tzz** [comment]
Are all the containers running, are there any errors in the browser or in the containers' logs?

**Reply by RicoSaupeB** [reply]
I just upgrade from 1.36.0 to 1.37.1. just a regular docker down, pull and up and now i am facing the same issue. the homepage still showing the thumbnails but all other pages dont load or when in develer tools just reloading in a loop. 

even pages like /admin/users
or if I click on a single image

btw. images are located in an external library, nothing changed there

<img width="1877" height="392" alt="image" src="https://github.com/user-attachments/assets/d23e5b74-302c-4b6f-9de1-5da463929634" />

**Reply by RicoSaupeB** [reply]
One update. It only happens when I call with my local domain (was working before the update), using the IP  address it works as expected.

**Reply by pserey** [reply]
Hello! Did you get any updates on this? I've performed the same upgrade (1.36.0 to 1.37.1) and I'm experiencing the same behavior. Same errors, but only at my local domain. "Solved" it by accessing it through the IP, but for external access, this will not work.

I've got more console errors before the "Failed to load module script...", here's the screenshot of my console during the reload loop:
<img width="836" height="292" alt="image" src="https://github.com/user-attachments/assets/b9d87412-a0bc-4e7b-863b-2152b377fbf7" />

**Reply by RicoSaupeB** [reply]
For me, i left the machine for some time and i came back to the portal 1 hour later and I could access the images and videos again. I am totally unclear why but happen it resolved itself. Maybe some kind of caching.
